frontend/ui/widget/checkbutton.lua

```diff
@@ -55,6 +55,7 @@ function CheckButton:initCheckButton(checked)
     self._textwidget = TextWidget:new{
         text = self.text,
         face = self.face,
+        max_width = self.max_width,
```

Comment:
Can you add `max_width = nil,` in the `local CheckButton = InputContainer:new{}` (after `toggle_text = nil,`), so it documents that this property is supported (I always look at that when I need a widget to know what I can do with it :)

Reply:
Of course :)